doc/api/assert.md

```diff
@@ -109,13 +109,11 @@ parameter is an instance of an `Error` then it will be thrown instead of the
 <!-- YAML
 added: v1.2.0
 changes:
-  - version: REPLACEME
+  - version: v8.5.0
```

Comment:
This should be kept as `REPLACEME` because it is a semver-major change.

Reply:
ok